### PR TITLE
Validation Badge path returns 404

### PR DESCRIPTION
**Pull request by MaxTCodes**
There was the error with the validation badge on the ReadMe, so for right now I removed it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 [![made-with-python](https://img.shields.io/badge/Made%20with-Python%203.8+-ffe900.svg?longCache=true&style=flat-square&colorB=00a1ff&logo=python&logoColor=88889e)](https://www.python.org/)
 [![AGPL](https://img.shields.io/badge/Licensed%20under-AGPL-red.svg?style=flat-square)](./LICENSE)
-[![Validation](https://github.com/Scoder12/rickchurch/actions/workflows/validation.yaml/badge.svg)](https://github.com/Scoder12/rickchurch/actions/workflows/validation.yaml)
```
Comment (ItsDrike) [inline README.md:5]: It was done for the old repo, I forgot to update it, it should just be a matter of changing the url to this one, workflow name is the same

**Review by ItsDrike — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MaxTCodes) [reply README.md:5]: I already tried to change just the GitHub username, and the badge still wasn't appearing. Ill double check that though!

Comment (ItsDrike) [reply README.md:5]: If not, just go to actions, find validation workflow and click create status badge, but it should work just with the url change

**Review by ItsDrike — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MaxTCodes) [reply README.md:5]: Yep, it just worked! I just made a new commit containing the fixed version!

**Pull request merged**